src/Umbraco.Web.UI.Client/src/packages/core/property/components/property-layout/property-layout.element.ts

```diff
@@ -104,17 +104,17 @@ export class UmbPropertyLayoutElement extends UmbLitElement {
 				display: grid;
 				grid-template-columns: 200px minmax(0, 1fr);
 				column-gap: var(--uui-size-layout-2);
-				border-bottom: 1px solid var(--uui-color-divider);
 				padding: var(--uui-size-layout-1) 0;
 			}
 
-			:host(:last-of-type) {
-				border-bottom: none;
-			}
-
 			:host > div {
 				grid-column: span 2;
 			}
+
+			:host([orientation='vertical']) {
+				padding: var(--uui-size-space-3) 0;
```

Comment:
This will impact all property layouts, also the ones outside this modal. The padding property is directly on the host element, so it should be possible to move this style change to the property-workspace-view-settings.element.ts file instead.